examples/local/vtorc-config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Debug": true,
+  "MySQLTopologyUser": "orc_client_user",
+  "MySQLTopologyPassword": "orc_client_user_password",
+  "MySQLReplicaUser": "vt_repl",
+  "MySQLReplicaPassword": "",
+  "BackendDB": "sqlite",
+  "SQLite3DataFile": "/tmp/vtorc.sqlite3",
```

Comment:
sqlite parameters need not be specified. They're currently defaulted to the correct value. Once we're sure we won't need them, we'll deprecate them. Actually, you may just be able to reuse https://github.com/vitessio/vitess/blob/master/config/orchestrator/default.json.